ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/dot1x.md

```diff
@@ -111,4 +127,46 @@
 
         # Interval period in seconds.
         accounting_update_interval: <int; 5-65535>
+
+      # Web authentication feature authenticates a supplicant through a web page, referred to as a captive portal.
+      captive_portal:
+
+        # The `enabled` key is mutually exclusive with `url` and `ssl_profile`.
```

Comment:
This is no longer true. But we should made `enabled` a required field.

Reply:
Fixed